client/sttp-client/src/main/scalajvm/sttp/tapir/client/sttp/SttpClientInterpreterPlatformSpecific.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.tapir.client.sttp
+
+import sttp.client3.HttpURLConnectionBackend
+import sttp.model.Uri
+import sttp.tapir.Endpoint
+
+trait SttpClientInterpreterPlatformSpecific {
+
+  def toBlockingClientUnsafe[I, E, O](e: Endpoint[I, E, O, Any], baseUri: Uri)(implicit
+      clientOptions: SttpClientOptions
+  ): I => Either[E, O] = {
+    val backend = HttpURLConnectionBackend()
+    val req = new EndpointToSttpClient(clientOptions, WebSocketToPipe.webSocketsNotSupportedForAny).toSttpRequestUnsafe(e, baseUri)
+    (i: I) => backend.send(req(i)).body
+  }
+
+  def toYoloClient[I, E, O](e: Endpoint[I, E, O, Any], baseUri: Uri)(implicit
```

Comment:
And finally also naming (it's hard ;) ). I'm not sure `yolo` is a good one ... maybe `toQuickClientThrowErrors`? Similarly, we could have a `toClientThrowErrors: I => F[O]`